README.md

```diff
@@ -1 +1,24 @@
-# officer-delta-processor
\ No newline at end of file
+# officer-delta-processor
+
+The Officer Delta Processor is responsible for reading from the officer-delta kafka topic.
```

Comment:
See
https://github.com/companieshouse/officer-delta-processor/pull/2/files#r673005072

Reply:
Changed to TBC as we don't have a high level design yet.